client/src/pages/Home.jsx

```diff
@@ -124,6 +125,10 @@ const Home = ({ code }) => {
     setDeleteStatus(false);
   };
 
+  const negate = (condition, shouldNegate) => {
+    return shouldNegate ? condition : !condition;
+  };
```

Comment:
why don't you just call this function if `shouldNegate` is true rather than passing it in ,,, seems pretty weird because otherwise the function just returns the same value??